### DISCUSSION TITLE
Word-wise detection handwritten

### DISCUSSION THREAD

**Discussion by PxCPeter2801** [PaddlePaddle/PaddleOCR#12685] [Q&A] [answered]
I have a dataset handwritten had annotated, and I used PPOCRLabel.py to test detect capabilities. I met 2 problems:
- How to rate the efficiency of the model when PPOCRLabel detects a line ( ex: 1 line includes 10 words, it is detected as 2 bounding boxes instead of 1 bounding box). So I think detection word-wise (1 word 1 box)  is more efficient than line-wise
- How I change the model is used in PPOCRLabel.py, i want to apply my own model trained with handwritten

**Comment by tink2123** [comment, marked as answer]
1. The default detection model is line-level. If you want to use word-level detection, you can change the English detection model.
2. For instructions on changing models, please refer to the documentation: https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/PPOCRLabel/README.md#32-built-in-model

**Comment by PxCPeter2801** [comment]
I met  a problem when auto recognition images handwritten English. Image and bounding box look like rotation 90 degrees. This case met some images, not at all
![1234](https://github.com/PaddlePaddle/PaddleOCR/assets/109299973/f1da4dc4-7c08-4b20-9828-4d21a4037506)

**Reply by SWHL** [reply]
This is due to an error in recognizing the direction of the current text line
